src/components/Charts/Difficulty.tsx

```diff
@@ -8,15 +8,15 @@ type Props = {
 }
 
 export default function Difficulty({ data }: Props): JSX.Element {
-  const yAccessor = React.useCallback((d: Metric) => d.average_difficulty / 1000000000, [])
+  const yAccessor = React.useCallback((d: Metric) => d.average_difficulty / 1000000000000, [])
```

Comment:
Nit, but this seems like a good place to use scientific notation (as I cannot easily scan this):

```suggestion
  const yAccessor = React.useCallback((d: Metric) => d.average_difficulty / 1e12, [])
```

Reply:
true 👍